kura/distrib/src/main/ant/build_equinox_distrib.xml

```diff
@@ -301,6 +302,10 @@ fi]]>
 		<replaceregexp file="${project.build.directory}/${build.output.name}/kura.init.raspbian" match="INSTALL_DIR=.*" replace="INSTALL_DIR=${kura.install.dir}" />
 		<replaceregexp file="${project.build.directory}/${build.output.name}/kura.service.edison" match="INSTALL_DIR" replace="${kura.install.dir}" />
 
+		<script language="javascript">
+	    var nativeTag = project.getProperty("native.tag");
+	    project.setNewProperty("org.eclipse.linux.usb.native.version", project.getProperty("org.eclipse.linux.usb." + nativeTag + ".version" ) );
```

Comment:
I tried to build the code, but Kura didn't started. I've found that this setNewProperty was not correctly working and the config.ini file was trying to start a bundle with version ${org.eclipse.linux.usb.native.version}